docs/content/preview/reference/drivers/go/yb-pgx-reference.md

```diff
@@ -85,51 +102,35 @@ The following table describes the connection parameters required to connect to t
 | database | Database name | yugabyte
 | user | User connecting to the database | yugabyte
 | password | Password for the user | yugabyte
-| load_balance | enables uniform load balancing | true
 
-To enable the cluster-aware connection load balancing, provide the parameter `load_balance=true` in the connection string as follows:
+#### Load balancing connection properties
 
-```go
-"postgres://username:password@localhost:5433/database_name?load_balance=true"
-```
+The following connection properties need to be added to enable load balancing:
 
-- It is **topology-aware**, which is essential for geographically-distributed applications.
+- load-balance - enable cluster-aware load balancing by setting this property to `true`; disabled by default.
+- topology-keys - provide comma-separated geo-location values to enable topology-aware load balancing. Geo-locations can be provided as `cloud:region:zone`.
```

Comment:
The format is `cloud.region.zone` (`.` instead of `:`)

Reply:
fixed it on both go and java pages.